src/__tests__/components/groupCard.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Får feilmelding på testen for denne komponenten 

![image](https://user-images.githubusercontent.com/32167479/97853018-d28eb600-1cf7-11eb-9e0b-3a3eae7a6b9e.png)
![image](https://user-images.githubusercontent.com/32167479/97853074-e76b4980-1cf7-11eb-873c-a0fa64656525.png)
![image](https://user-images.githubusercontent.com/32167479/97853092-ef2aee00-1cf7-11eb-8acb-27064676a3ad.png)

Reply:
Så at jeg kommenterte på selve filen og ikke på testfilen. Feilen ligger i groupCard.test.tsx, virker det som :)

Reply:
Feilen er i fetchDetailThunk.test.ts, har glemt å oppdatere action type etter FETCH_SUCCESS, push med fix kommer om 5